server/graphql/models/User/mutation.js

```diff
@@ -20,12 +20,13 @@ const InputUser = new GraphQLInputObjectType({
   description: 'The user account',
   fields: () => ({
     id: {type: new GraphQLNonNull(GraphQLID), description: 'The user UUID'},
-    email: {type: new GraphQLNonNull(GraphQLEmail), description: 'The user email'},
-    handle: {type: new GraphQLNonNull(GraphQLString), description: 'The user handle'},
-    name: {type: new GraphQLNonNull(GraphQLString), description: 'The user name'},
+    email: {type: GraphQLEmail, description: 'The user email'},
+    handle: {type: GraphQLString, description: 'The user handle'},
+    name: {type: GraphQLString, description: 'The user name'},
```

Comment:
These fields are required for all user accounts, so let's keep the `new GraphQLNonNull(...)` wrapping.